projects/storefrontapp-e2e-cypress/cypress/helpers/coupons/my-coupons.ts

```diff
@@ -100,6 +116,7 @@ export function claimCoupon(couponCode: string) {
     'getClaimedCouponPage'
   );
 
+  //TODO when 'enableClaimCustomerCouponWithCodeInRequestBody' is true, call the mothod of WithCodeInBodywaitForClaimCoupon instead once ClaimCustomerCouponWithCodeInBody works in the backend(available since Commerce 2211.28)
```

Comment:
hi @rmch91  , not sure whether you are enjoying the holiday, hope not bother you. As I already disable the toggle to avoid break CI because of deployed commerce version issue. And I skip new added e2e test, but for the existed test cases, I left comments like this. Because when enable 'enableClaimCustomerCouponWithCodeInRequestBody', the case will failed because it need check new occ endpoints calling , so we need update the case(update method check in code) here after enable 'enableClaimCustomerCouponWithCodeInRequestBody', do you think the left comments is enough for future update? or is there other steps or ideas for updating cases? creating jira tickets to follow it?
Thanks.